dev/io.openliberty.webcontainer.servlet.6.0.internal/bnd.bnd

```diff
@@ -0,0 +1,64 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: WAS WebContainer
+Bundle-SymbolicName: io.openliberty.webcontainer.servlet.6.0.internal
+Bundle-Description: WAS WebContainer with Servlet 6.0 support
+WLP-ServerName: IBM WebSphere Application Server
+WLP-ServerVersion: ${liberty.beta.version}
+
+javac.source: 11
+javac.target: 11
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+Export-Package: \
+    io.openliberty.webcontainer60.internal.session.impl*;provide:=true
```

Comment:
If it is exported it shouldn't have `internal` in the package name.

Reply:
Is it possible to combine this bundle with the factories bundle?

Reply:
@tevans78 can correct me though.  `internal` should only be used for packages that are not exported from a bundle right?

Reply:
Regarding combining we don't want to do that. The factories bundle is where `com.ibm.ws.webcontainer.speclevel` exists. There is a properties file that allows us to determine which version of the Servlet specification is in use in our common webcontainer code. Only one factories bundle is loaded per server feature. However, the non factories servlet spec specific bundles (serv.et.3.1, servlet.4.0, servlet.6.0) can be loaded by more than one feature.

Reply:
Exported packages still have internal in them.  Can fix this in a follow on PR since it is servlet 6.0 specific code.  This makes things messy for io.openliberty.servlet.6.0.internal.factories/bnd.bnd leaving it with internal in the package name.

Reply:
Removed .internal. in 2 packages: io.openliberty.webcontainer60.internal.session.impl -> io.openliberty.webcontainer60.session.impl and io.openliberty.session.internal - > io.openliberty.session.impl

Reply:
Not sure if both were needed to be done.  Were both imported by other bundles?  I will leave it as it is right now and move forward.